Remove old behaviors descriptions and references (#14)
apptainer/apptainer-userdocs: security_options.rst

 Security related action options
*********************************

> {Singularity} counts in with some flags that can be passed to the

Reviewer: ```suggestion
{Singularity} has many flags that can be passed to the
```
